java-squid/src/main/java/org/sonar/java/se/constraint/ObjectConstraint.java

```diff
@@ -45,11 +44,19 @@ public ObjectConstraint(boolean isNull, boolean disposable, @Nullable Tree synta
     this.status = status;
   }
 
+  public static ObjectConstraint nullConstraint() {
+    return nullConstraint(null);
+  }
+
+  public static ObjectConstraint nullConstraint(@Nullable  Tree syntaxNode) {
+    return new ObjectConstraint(true, false, syntaxNode, null);
+  }
+
   public ObjectConstraint inverse() {
-    if (isNull) {
+//    if (isNull) {
       return new ObjectConstraint(!isNull, disposable, syntaxNode, status);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AIndentationCheck)